### DISCUSSION TITLE
I am very confused about hardware and how to set up a project...can someone update the ultimate hyperhdr guide?

### DISCUSSION THREAD

**Discussion by X-15** [awawa-dev/HyperSerialESP32#58] [Q&A]
Hello everyone,

I've been reading the 2023 Ultimate Guide multiple times, but I'm still confused about how to proceed. It seems like many things have changed, including the recommendations, leaving me feeling quite lost. I really need a straightforward guide on what to buy and how to solder (if needed). While the Ultimate Guide explains how everything works or doesn't, I can't find a practical, hands-on guide to help me build the hardware from scratch.

Currently, my Windows PC is my video source, and I've installed HyperHDR. I've successfully synced it to my Philips Hue Bridge, which allows the lights to follow my TV output. This setup works, but now I'd like to add an LED strip to my 85" TV, connecting the strip to a controller and Raspberry Pi. Initially, I want to try running it via Wi-Fi to see its performance. If it's not up to par, I can use a USB grabber-HDMI splitter, whichever is currently recommended which I also don't know if I want to keep 4k and all the fancy hdmi features.

Unfortunately, I haven't found a comprehensive, easy-to-follow guide for this. Could anyone kindly provide direct instructions? I don't want to rely on guesswork or assume a product will work because it seems logical on paper. I also don't have the patience or expertise to guess about soldering correctly.

I would sincerely appreciate any assistance you can provide.

Thank you so much!

**Comment by awawa-dev** [comment]
Hi
The only major thing that has come out since the last guide and has changed things a bit (simplified setup since some rp2040s have a built-in level shifter so it's ideal if you dont have a patience) is HyperSerialPico but the guide mention about it. More details can be found on HyperSerialPico project and it's very similar to HyperSerialESP32 only rp2040 is better and easier to flash than esp32. The rest is valid and very detailed described. The most frequently chosen configurations when it comes to the grabber are ezcoo splitter + ms2109 (if we only have USB2.0) or ms2130 (usb3.0). Or Elgato HD60x as all in one solution (if we do not need HDCP and Dolby Vision LLDV support). There are many similar topics on HyperHDR project discussion session.
